### ISSUE TITLE
Live connection test is flaky, blocking several PRs

### ISSUE THREAD

**Issue by erichiggins** [firebase/emberfire#534]
The [`live firebase connection` acceptance test](https://github.com/firebase/emberfire/blob/7fbe8501062beb3f6b44ac55313802724d164d87/tests/acceptance/live-connection-test.js#L12) (authored by @steven-ferguson ) is timing out and blocking contributor PRs.  It'd be helpful if the maintainers of this repo could either correct or fix this tests so that contributions aren't blocked or delayed.

Affected PRs:

* #533 
* #528 
* #527 

Error message from Travis CI:

```
not ok 22 Firefox 51.0 - Acceptance: live firebase connection handles live connections in testing
    ---
        message: >
            timeout of 2000ms exceeded. Ensure the done() callback is being called in this test.
        stack: >
            [35]</</Runnable.prototype.resetTimeout/this.timer<@http://localhost:7357/assets/test-support.js:6326:19
            
        Log: |
    ...
```

**Comment by blkeller** [comment]
I retriggered Travis builds a few times in PR #533 (with identical code each time), just to see if I was unlucky on a given day, but no dice.

Looking back at the build history, it's very inconsistent as to whether the failure occurs or not. Over the course of three build events, test suites with identical parameters have passed at some times and failed at others.  I have yet to get lucky enough for every test suite in a build to pass, though I'm guessing that if I tried for long enough, I would eventually succeed.

Links to my build history:
- [First build attempt](https://travis-ci.org/firebase/emberfire/builds/305506806)
- [Second build attempt](https://travis-ci.org/firebase/emberfire/builds/305515351)
- [Third build attempt](https://travis-ci.org/firebase/emberfire/builds/315516771)

**Comment by erichiggins** [comment]
Thanks for the additional info @blkeller !

**Comment by erichiggins** [comment]
Pinging @tstirrat and @aputinski, who appear to be the historically most active contributors, to see if they can help.
Is this project dead? No commits since August?

**Comment by jamesdaniels** [comment]
@erichiggins not dead, undergoing a rewrite. Will have more to announce soon.

**Comment by erichiggins** [comment]
@jamesdaniels That's great to hear!  Any chance we can get these PRs unblocked in the meantime?

**Comment by jamesdaniels** [comment]
Closed with #538, switching to Chrome for the travis tests rather than Firefox. Will see if that unblocks the PRs.

**Comment by erichiggins** [comment]
Possibly related: I just started doing some testing of my emberfire-powered web app in Firefox and noticed that there may actually be an issue with the live connection between Firefox and Firebase.

For example, one view in my web app allows a user to upload a file to Firebase Storage. Firebase Functions are then used to process the file and make entries into the Realtime Database. 

In Firefox, the list view of those database results is **not** updated automatically like it is in Chrome. Instead, I have to navigate to a different view then back, or refresh the page entirely.

If you think this issue is worth investigating, I can file it under a new ticket. Just wanted to mention it here first in case it's actually related to the failed tests which used Firefox.

**Comment by jamesdaniels** [comment]
Yes @erichiggins , please file that as a ticket + note the version of the Firebase JS SDK that you're using. It could be something that was patched upstream in a later version then we are using here.
